packages/desktop/components/popups/send/SendFormPopup.svelte

```diff
@@ -68,6 +68,10 @@
     $: hasSpendableNfts = $ownedNfts.some((nft) => nft.isSpendable)
     $: isLayer2 = !!iscpChainAddress
     $: isSendTokenTab = activeTab === SendForm.SendToken
+    $: isBaseToken = getBaseToken().unit === getUnitFromTokenMetadata(transactionDetails.tokenMetadata)
```

Comment:
why not using the token standard?
```suggestion
    $: isBaseToken =
        transactionDetails.type === NewTransactionType.TokenTransfer &&
        transactionDetails?.asset?.metadata?.standard === TokenStandard.BaseToken
```